src/CMakeLists.txt

```diff
@@ -72,7 +72,7 @@ function(gz_msgs_protoc)
     # Full path to generated header (${PROJECT_BINARY_DIR}/include/gz/msgs/foo.pb.h)
     set(output_header "${gz_msgs_protoc_OUTPUT_CPP_DIR}${proto_package_dir}/${FIL_WE}.pb.h")
     # Full path to generated detail header (${PROJECT_BINARY_DIR}/include/gz/msgs/detail/foo.pb.h)
```

Comment:
`details` in this comment as well?

Reply:
[bec5cd9](https://github.com/gazebosim/gz-msgs/pull/304/commits/bec5cd9440985441fb315f4781ef6f1b273236af)